src/main/java/org/mockito/internal/progress/MockingProgressImpl.java

```diff
@@ -111,6 +115,17 @@ public void stubbingCompleted(Invocation invocation) {
         stubbingInProgress = null;
         getStubbingListener().newStubbing(invocation);
     }
+
+    /**
```

Comment:
With 2 stars this comment appear as a javadoc comment of the above method (`toString`)